Firebase/Auth/Source/Public/FIRAuth.h

```diff
@@ -426,11 +426,12 @@ NS_SWIFT_NAME(Auth)
                             completion:(nullable FIRAuthDataResultCallback)completion;
 
 /** @fn signInWithCredential:completion:
-    @brief Convenience method for `signInAndRetrieveDataWithCredential:completion:` This method
+    @brief Please use `signInAndRetrieveDataWithCredential:completion:` instead. This method
         doesn't return additional identity provider data.
  */
 - (void)signInWithCredential:(FIRAuthCredential *)credential
-                  completion:(nullable FIRAuthResultCallback)completion;
+                  completion:(nullable FIRAuthResultCallback)completion __attribute__((deprecated));
```

Comment:
Nit: Please use `DEPRECATED_MSG_ATTRIBUTE("<Message Here>")` instead so the users get a warning in Xcode.

Reply:
It seems like "__attribute__((deprecated))" also results in a warning if the deprecated API is used. Is this that the concern you had? Whether the warming message would be shown?

Reply:
Done.